helper/imagemin.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+module.exports = function(gulp, plugins, config, name) { // eslint-disable-line func-names
+  // Return function that is executed inside of .pipe()
+  return () => {
+    const theme          = config.themes[name],
+          srcBase        = config.projectPath + theme.src,
+
+          // Look for any images in the web/images directory and minify them in place
+          imageDir       = srcBase + '/web/images/**/*',
+          minnedImageDir = srcBase + '/web/images/';
```

Comment:
Do you think it's worth running this on `/web/icons` too? I'm not sure how/if that'll impact the icon task... I seem to recall that gulpicon can only optimise PNGs.